Molly/README.md

```diff
@@ -0,0 +1,168 @@
+
+**d)** `x == (x+'')` True. Both side are false, which makes them equal--thereby making this statement true.
+
+**e)** `'' == ' '` True. Both sides are false-y, which makes the statement true
+
+**f)** `x = true` True. We have assigned the value true to x, so the statement will come out as true.
+
+**g)** `var x; x == 'undefined'` False. x is undefined and therefore false-y while 'undefined' is a string and truth-y, which makes the two sides unequal.
+
+**h)** `'9'<'10'` True. The number 9 is less than the number 10. (I checked this in the console and know it's incorrect, but I can't figure out why.)
+
+**i)** `typeof x + 1 === "number"` False. The left side is undefined and the right side is a string.
+
+**j)** `typeof x % 2 === "number"` Fales. Left side is NaN and the right side is a string, so they are not the same.
+
+**k)** `typeof (x % 2) === "number"` True, but I can't explain why. Somehow the parenthesis fixes things.
+
+**l)** `x++ == ++x` False, but I can't explain why.
+
+**m)** `++x == x++` True, but I can't explain why.
+
+**n)** `"1"+x == 1+x` False. x is undefined and therefore false-y.
+
+**o)** `"0"+1 == 1` True. 0 is a real number, so the computer adds it to 1.
+
+**p)** `(typeof (x+1))===(typeof x)` False. The left side is a number and the right side is undefined.	
+
+**q)** `(x*1 == x) || ((typeof x) != "number")` True. The left side is undefined and false-y, but the right it truth-y.
+
+**r)** `(x=(typeof (x+(typeof x))))==x` True. Isn't this just asking if x==x?
+
+---
+
+All of the following can be solved with ordinary expressions and global variables with primitive values.  You don't need functions, loops, or other topics beyond our first two classes._
+
+ **2)** (_10%_)
+
+Assume variables x, y, and z are numbers.
+
+**a)**
+Write an expression for the mean (i.e. average) of x, y, and z.
+
+var mean = (x + y + z) / 3;
+
+**b)**
+Write a series of expressions to adjust each of x, y, and z halfway toward the mean of the three.
+That is, reset the value of each variable to something new based on its previous value.
+
+var newMean = ((x/2) + (y/2) + (z/2)) / 3;
+
+---
+
+**3)** (_20%_)
+
+Suppose you're encoding geometric shapes in a Cartesian (2D) coordinate system, and you represent a rectangle with four numeric variables:
+
+- _l_ : horizontal position of left edge (relative to some origin);
+- _r_ : horizontal position of right edge;
+- _t_ : vertical position of top edge;
+- _b_ : vertical position of bottom edge.
+
+**a)**
+Write an expression for the rectangle's area.
+
+var width = (r-l);
+var height = (t-b);
+var area = width * height;
+
+**b)**
+Write an expression which is true if the rectangle is taller than it is wide, and false otherwise.
+
+if (height > width) {
+	true;
+} else {
+	false;
+}
+
+**c)**
+Write an expression for the circumference of the biggest circle which can fit inside the rectangle.  (Hint: you'll need logic similar to that in **b**.)
+
+var cir = 3.14 * width;
+
```

Comment:
Good start! We were looking for something like this:
var smallerDiam;
var isTall = (t-b)>(r-l);
if (isTall) {
    smallerDiam = r-l;
} else {
    smallerDiam = t-b;
//OR:
var smallerDiam = isTall? (r-l): (t-b);

//OR:

var smallerDiam = (r-l)*isTall + (t-b)*(!isTall);
var circumference = smallerDiam * Math.PI;